aimsprop/plot.py

```diff
@@ -234,6 +234,10 @@ def plot_population(
 
     # Spaghetti plots from bundles
     Imap = {I: Iind for Iind, I in enumerate(bundle.Is)}
+    for bundle2 in bundles:
```

Comment:
@hayleyweir can you confirm this will maintain the intended plotting behavior already in the package?